<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newPass</class>
 <widget class="QDialog" name="newPass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>682</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create a New Password</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>630</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>341</width>
     <height>411</height>
    </rect>
   </property>
   <property name="title">
    <string>Password details</string>
   </property>
   <widget class="QLineEdit" name="others">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>270</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Disallowed Characters:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Required Password Length (6-99):</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="lowercase">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Lowercase Chars</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="integers">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Digits (0-9)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="uppercase">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Uppercase Chars</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="punctuation">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Symbols </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Other exclusions: </string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="numChars">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>6</number>
    </property>
    <property name="value">
     <number>12</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>220</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string> (!&quot;#$%&amp;'()*+, -./:;&lt;=&gt;?@[]^_`{|}~)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>271</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Disallowed/excluded characters are characters that will not be selected in the password creation process.</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>440</y>
     <width>341</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>User data</string>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Website/application name:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sitename">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Username: </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="user_name">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>newPass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>newPass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
